src/recipe.cpp

```diff
@@ -680,9 +681,13 @@ std::string recipe::missing_proficiencies_string( Character *c ) const
     }
 
     std::string color = "yellow";
+    std::string name_color = "cyan";
     std::string missing = enumerate_as_string( missing_profs.begin(),
     missing_profs.end(), [&]( const prof_penalty & prof ) {
-        return profstring( prof, color );
+        if( !c->has_prof_prereqs( prof.id ) ) {
+            name_color = "red";
+        }
+        return profstring( prof, color, name_color );
```

Comment:
This makes `name_color` red for all subsequent proficiencies regardless of mastered or not. Did you want to express
```suggestion
        return profstring( prof, color, c->has_prof_prereqs( prof.id ) ? "cyan" : "red" );
```

Reply:
thank you, should be fixed now.